framework/src/wifi_manager/wifi_manager_dhcpc.c

```diff
@@ -22,11 +22,14 @@
 #include <string.h>
 #include <sys/types.h>
 #include <netutils/netlib.h>
+#include <tinyara/net/netlog.h>
 #include <wifi_manager/wifi_manager.h>
 #include "wifi_manager_utils.h"
 #include "wifi_manager_error.h"
 #include "wifi_manager_dhcp.h"
 
+#define TAG "[WM]"
```

Comment:
How about change [DHCPC]?

Reply:
it's wifi_manager module so I think wm is better.